src/components/v5/common/Pills/types.ts

```diff
@@ -0,0 +1,41 @@
+import { MessageDescriptor } from 'react-intl';
+import { SimpleMessageValues } from '~types';
+
+export type ExtensionStatusBadgeMode =
```

Comment:
I think it can be `export type ExtensionStatusBadgeMode extends Pick<PillMode, 'new and other repeatable values in PillMode'> {}`